website/source/docs/providers/aws/r/s3_bucket_notification.html.markdown

```diff
@@ -0,0 +1,170 @@
+---
+layout: "aws"
+page_title: "AWS: aws_s3_bucket_notification"
+side_bar_current: "docs-aws-resource-s3-bucket-notification"
+description: |-
+  Provides a S3 bucket notification resource.
+---
+
+# aws\_s3\_bucket\_notification
+
+Provides a S3 bucket notification resource.
+
+## Example Usage
+
+### Add notification configuration to SNS Topic
+
+```
+resource "aws_sns_topic" "topic" {
+    name = "s3-event-notification-topic"
+    policy = <<POLICY
+{
+    "Version":"2012-10-17",
+    "Statement":[{
+        "Effect": "Allow",
+        "Principal": {"AWS":"*"},
+        "Action": "SNS:Publish",
+        "Resource": "arn:aws:sns:*:*:terraform-test-topic",
```

Comment:
This `terraform-test-topic` needs to match up to a valid SNS Topic name, correct? The example config does not complete successfully: 

```
* aws_s3_bucket_notification.bucket_notification: Error putting S3 notification configuration: InvalidArgument: Unable to validate the following destination configurations
```

Changing it to `s3-event-notification-topic` should work